sources/api/schnauzer/src/lib.rs

```diff
@@ -137,6 +137,7 @@ pub fn build_template_registry() -> Result<handlebars::Handlebars<'static>> {
     );
     template_registry.register_helper("localhost_aliases", Box::new(helpers::localhost_aliases));
     template_registry.register_helper("etc_hosts_entries", Box::new(helpers::etc_hosts_entries));
+    template_registry.register_helper("any-enabled", Box::new(helpers::any_enabled));
```

Comment:
Oh man sorry, this is a mess. Above we added this comment

```
// Prefer snake case for helper names (we accidentally created a few with kabob case)
```

Should we follow it and change this to snake_case?

```suggestion
    template_registry.register_helper("any_enabled", Box::new(helpers::any_enabled));
```

Reply:
I completely missed that comment and had thought I was keeping things more consistent by going with kabob case. Updating to snake case.